utils/index.js

```diff
@@ -7,6 +7,9 @@ export { focus };
 export { keycodes };
 export { decodeEntities };
 
+// Keep all old wp.utils components
```

Comment:
Too bad it can't be done programmatically, at least without reverting the entire file to CommonJS. I expect you may have encountered this too. Worth a comment to stop those who'd be inclined to try to make the same optimization?

Reply:
You mean something like this right? `export { ...wp.utils }`

Reply:
Yeah, if it would work. I even tried falling back to CommonJS:

```js
for ( const key in wp.utils ) {
	if ( wp.utils.hasOwnProperty( key ) ) {
		exports[ key ] = wp.utils[ key ];
	}
}
```

https://github.com/webpack/webpack/issues/4039

Reply:
Doesn't seem to work, I added a comment.